components/apimgt/org.wso2.carbon.apimgt.core/src/test/java/org/wso2/carbon/apimgt/core/dao/impl/ApiDAOImplIT.java

```diff
@@ -596,10 +596,10 @@ public void testGetAPIsByStatus() throws Exception {
             apiDAO.addAPI(api);
         }
 
-        List<API> blockedAPIsSummary = new ArrayList<>();
-        for (int i = 0; i < numberOfBlocked; ++i) {
-            API api = SampleTestObjectCreator.createUniqueAPI().lifeCycleStatus(blockedStatus).build();
-            blockedAPIsSummary.add(SampleTestObjectCreator.getSummaryFromAPI(api));
+        List<API> maintenanceAPIsSummary = new ArrayList<>();
+        for (int i = 0; i < numberOfInMaintenance; ++i) {
```

Comment:
Yes, it will be useful if the limit value in the for loop is changed for different test, but in this case it is hard code as 1 in this test method. IMO when the requirement comes, we can add the loop else it is not required.